docs/review_process.md

```diff
@@ -122,7 +122,7 @@ It doesn't always mean accepting all the suggestions, but at least providing a r
 
 ## Automatic Merges
 
-The bot runs Automatic Merges every 30 minutes. Currently, it can only merge a single PR in this timeframe, so there is a theoretical limit of 48 PRs merged per day (in practice, it's even less for reasons listed below).
+The bot runs Automatic Merges every 20 minutes. Currently, it can only merge a single PR in this timeframe, so there is a theoretical limit of 48 PRs merged per day (in practice, it's even less for reasons listed below).
```

Comment:
```suggestion
The bot runs Automatic Merges several times every hour. It merges a single PR in each run, so there is a limit of PRs merged per day. The frequency can be manually increased or decreased depending on the number of PRs that accumulate in the queue.
```

Reply:
I prefer keep it explicit in terms of time. We have some users which ask the interval so we just need to point to this link. It's true we need to update it manually, but the information is transparent.

Reply:
yes, explicit is better